src/status_im/bots/events.cljs

```diff
@@ -88,19 +88,19 @@
 ;;;; Handlers
 
 (handlers/register-handler-fx
-  :set-in-bot-db
-  [re-frame/trim-v]
-  (fn [{:keys [db]} [params]]
-    (set-in-bot-db db params)))
+ :set-in-bot-db
```

Comment:
shouldn't these be 2 spaces according to https://github.com/bbatsov/clojure-style-guide#source-code-layout--organization ? That's what my clojure-mode auto-indents to as well

Reply:
fixed

Reply:
@yenda is it possible to run auto-indents for the whole project on your side just to see what is the difference?

Reply:
@rasom you could check with https://github.com/weavejester/cljfmt 
I believe cider-format-buffer uses same formatting rules.

Reply:
I am not aware of an automated way I think I have to open and save each file one by one. I'll give 10 min of googling to see if there is some emacs magic to do that.

Reply:
I am not aware of an automated way I think I have to open and save each file one by one. I'll give 10 min of googling to see if there is some emacs magic to do that.

Reply:
@rasom just confirmed with cider author that it uses `cljfmt` internally, so we could run `lein cljfmt fix src/*`

Reply:
ok, i'll try this, thanks

Reply:
@yenda btw this is a function call so we should have only one space here